### PR TITLE
adding doctor feature

### PR DESCRIPTION
**Pull request by eugenchen**
+to do:
Consider using a rich text editor for the description to allow for formatting.
Image uploads should be optimized for faster loading times when patients view the doctor's profile.

### CODE REVIEW

File: src/pages/AddDoctorPage/AddDoctorPage.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Header from "../../components/Header/Header";
+import AddDoctorForm from "./AddDoctorForm";
+import Footer from "../../components/Footer/Footer";
+import { Link } from 'react-router-dom';
+
+const AddDoctorPage: React.FC = () => {
```
Comment (YuriiSkyba1) [inline src/pages/AddDoctorPage/AddDoctorPage.tsx:7]: please use functions without React like in another components

**Review by YuriiSkyba1 — CHANGES_REQUESTED** [review]

File: src/pages/AddDoctorPage/AddDoctorForm.tsx [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+    onSubmit: (values) => {
+      console.log(values);
+      setShowSuccessMessage(true);
+      formik.resetForm();
+    },
+  });
+  useEffect(() => {
+    if (showSuccessMessage) {
+      const timeout = setTimeout(() => {
+        setShowSuccessMessage(false);
```
Comment (dadclak) [inline src/pages/AddDoctorPage/AddDoctorForm.tsx:68]: why you don't use the toast for information for user?

File: src/pages/AddDoctorPage/AddDoctorForm.tsx [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+          <div className="relative">
+            <select
+              id="specialization"
+              name="specialization"
+              onChange={formik.handleChange}
+              onBlur={formik.handleBlur}
+              value={formik.values.specialization}
+              className="appearance-none w-full h-full pl-3 pr-8 bg-transparent border border-gray-300 rounded my_input"
+            >
+              <option value="" label="Select Specialization" />
```
Comment (dadclak) [inline src/pages/AddDoctorPage/AddDoctorForm.tsx]: you can generate this options by loop. And declare this data to const array then use here.

File: src/pages/LogSignPage/AuthorizationForm/AuthorizationForm.tsx [hunk 4]
```diff
@@ -1,7 +1,7 @@
 
 import '../../../assets/styles/Tailwind.css';
 import {useFormik} from "formik";
-import './AuthorizationForm.css';
+// import './AuthorizationForm.css';
```
Comment (dadclak) [inline src/pages/LogSignPage/AuthorizationForm/AuthorizationForm.tsx]: you can delete this row

**Comment by YuriiSkyba1** [comment]
please fix conflicts

File: src/pages/AddDoctorPage/AddDoctorForm.tsx (hunk 3 above)
Comment (eugenchen) [reply src/pages/AddDoctorPage/AddDoctorForm.tsx]: changed

File: src/pages/AddDoctorPage/AddDoctorPage.tsx (hunk 1 above)
Comment (eugenchen) [reply src/pages/AddDoctorPage/AddDoctorPage.tsx:7]: changed

File: src/pages/AddDoctorPage/AddDoctorForm.tsx (hunk 2 above)
Comment (eugenchen) [reply src/pages/AddDoctorPage/AddDoctorForm.tsx:68]: changed

File: src/pages/LogSignPage/AuthorizationForm/AuthorizationForm.tsx (hunk 4 above)
Comment (eugenchen) [reply src/pages/LogSignPage/AuthorizationForm/AuthorizationForm.tsx]: deleted

**Comment by eugenchen** [comment]
@YuriiSkyba1 fixed

**Review by YuriiSkyba1 — APPROVED** [review]

**Pull request merged**